src/Workspaces/CSharp/Portable/Formatting/CSharpFormattingOptions.cs

```diff
@@ -65,7 +65,7 @@ public static class CSharpFormattingOptions
 
         public static Option<bool> IndentSwitchSection { get; } = new Option<bool>(IndentFeatureName, "IndentSwitchSection", defaultValue: true);
 
-        public static Option<bool> IndentSwitchCaseSection { get; } = new Option<bool>(IndentFeatureName, "IndentSwitchCaseSection", defaultValue: true);
+        public static Option<SwitchCaseIndentOptions> IndentSwitchCaseSection { get; } = new Option<SwitchCaseIndentOptions>(IndentFeatureName, "IndentSwitchCaseSection", defaultValue: SwitchCaseIndentOptions.AlwaysIndent);
```

Comment:
I can do it that way if we want.  Would probably make sense given the breaking change comment below. I'll push another change today that will keep the original option and map to the new one.